pages/revoke.tsx

```diff
@@ -0,0 +1,10 @@
+import Allowances from 'components/Allowances';
+import { usePageView } from 'hooks/useAnalytics';
+import React, { memo } from 'react';
+
+const Revoke = () => {
+  usePageView('/revoke', 'Manage Allowances');
+  return <Allowances />;
+};
+
+export default memo(Revoke);
```

Comment:
`nit`: memo is not necessary as it has no props

Reply:
same as above